src/TypeScript/EslintBridgeClient/EslintBridgeHttpWrapper.cs

```diff
@@ -31,44 +31,55 @@ namespace SonarLint.VisualStudio.TypeScript.EslintBridgeClient
 {
     internal interface IEslintBridgeHttpWrapper : IDisposable
     {
-        Task<string> PostAsync(Uri serverEndpoint, object request, CancellationToken cancellationToken);
+        Task<string> PostAsync(string serverEndpoint, object request, CancellationToken cancellationToken);
     }
 
     internal sealed class EslintBridgeHttpWrapper : IEslintBridgeHttpWrapper
     {
         private readonly ILogger logger;
         private readonly HttpClient httpClient;
+        private readonly IEslintBridgeProcess eslintBridgeProcess;
 
-        public EslintBridgeHttpWrapper(ILogger logger)
-            : this(new HttpClientHandler(), logger)
+        public EslintBridgeHttpWrapper(IEslintBridgeProcessFactory eslintBridgeProcessFactory, ILogger logger)
```

Comment:
I decided to place the process handling here for now. I'm still not sure how the Client will behave once we add the typescript stuff (will we have 2 clients? probably). So this made it simpler to keep the changes in the client to a minimum. I will revisit this when working on TypescriptAnalyzer, unless you think it's better to tackle it now.

Reply:
I assumed we'd have two clients. A client that manages two processes seems unnecessarily complicated - the caller would always to specify which process it wanted to talk to.

Reply:
"init-linter" method aside, all the methods are either JS or TS, so it's possible to have one client that manages both. (I think it's better to have 2 clients though)